src/wallet/wallet.h

```diff
@@ -283,10 +142,10 @@ class ReserveDestination
 protected:
     //! The wallet to reserve from
     CWallet* pwallet;
+    //! The ScriptPubKeyMan to reserve from. Based on type when GetReservedDestination is called
+    ScriptPubKeyMan* m_spk_man;
```

Comment:
It'd be good to initialize this to nullptr since it's not set in the constructor like pwallet is

Reply:
Done